src/scss/modifiers/_inverse.scss

```
@@ -57,6 +58,15 @@
 				background-color: oColorsByName('white');
 			}
 		}
+
+		@if $file {
+			.o-forms-input--file ::file-selector-button {
+				@include oButtonsContent($opts: (
+					'type': 'primary',
+					'theme': 'inverse'
+				));
```

We don't need all the button styles, just to override the theme, and can save some bytes here :) 
```suggestion
				), $include-base-styles: false);
```